subprojects/core/src/main/java/org/gradle/api/internal/DelegatingDomainObjectSet.java

```diff
@@ -29,10 +29,10 @@
 import java.util.Set;
 
 public class DelegatingDomainObjectSet<T> implements DomainObjectSet<T> {
-    private final DomainObjectSet<T> backingSet;
+    protected final DomainObjectSet<T> delegate;
```

Comment:
nitpick: Is there a good reason why this is protected and not private with access through a getter?  I didn't see anything but maybe I overlooked it.

Reply:
No particular reason. I can change this.